src/models/BaseModel.py

```diff
@@ -75,12 +88,15 @@ def __init__(self, train_size=0.75, test_size=None, force_load_all=False):
                                                            train_size=train_size,
                                                            test_size=test_size,
                                                            random_state=1)
-
-        self.description_train_data, self.description_test_data = train_test_split(
-            self.description_feature,
-            train_size=train_size,
-            test_size=test_size,
-            random_state=1)
+        try:
+            self.description_train_data, self.description_test_data = train_test_split(
+                self.description_feature,
+                train_size=train_size,
+                test_size=test_size,
+                random_state=1)
+        except:
+            print("Not loading description training/test data because "
```

Comment:
https://github.com/soton-data-mining/job-salary-prediction/blob/1b75070a0ff83927312437bfc36119832eba55d3/src/models/BaseModel.py#L45-L53

when `force_load_all` is False